lib/config/optionsconfiguration.go

```diff
@@ -41,6 +41,7 @@ type OptionsConfiguration struct {
 	OverwriteRemoteDevNames bool     `xml:"overwriteRemoteDeviceNamesOnConnect" json:"overwriteRemoteDeviceNamesOnConnect" default:"false"`
 	TempIndexMinBlocks      int      `xml:"tempIndexMinBlocks" json:"tempIndexMinBlocks" default:"10"`
 	UnackedNotificationIDs  []string `xml:"unackedNotificationID" json:"unackedNotificationIDs"`
+	Fsync                   bool     `xml:"fsync" json:"fsync" default:"false"`
```

Comment:
I'd set this to true, unless @calmh disagrees?
Also, I am not sure if we even should do this, perhaps we should enable this by default, given it's nice a little price to pay in order to guarantee we don't corrupt someones files.